api/src/main/java/io/opentelemetry/trace/TracerFactory.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.trace;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * A factory for creating named {@link Tracer}s.
+ *
+ * @see io.opentelemetry.OpenTelemetry
+ * @see io.opentelemetry.trace.Tracer
+ * @since 0.1.0
+ */
+@ThreadSafe
+public interface TracerFactory {
+
+  /**
+   * Creates a new tracer instance.
+   *
+   * @param instrumentationName The name of the instrumentation library.
```

Comment:
Maybe we should add something like `NOT the name of the instrument*ed* library` to avoid confusion?

Reply:
Will update.

Reply:
done